pinot-controller/src/main/java/org/apache/pinot/controller/util/ServerSegmentMetadataReader.java

```diff
@@ -265,8 +284,12 @@ public List<ValidDocIdsMetadataInfo> getValidDocIdsMetadataFromServer(String tab
         List<ValidDocIdsMetadataInfo> validDocIdsMetadataInfoList =
             JsonUtils.stringToObject(validDocIdsMetadataList, new TypeReference<ArrayList<ValidDocIdsMetadataInfo>>() {
             });
-        for (ValidDocIdsMetadataInfo validDocIdsMetadataInfo : validDocIdsMetadataInfoList) {
-          validDocIdsMetadataInfos.put(validDocIdsMetadataInfo.getSegmentName(), validDocIdsMetadataInfo);
+        for (ValidDocIdsMetadataInfo validDocIdsMetadataInfo: validDocIdsMetadataInfoList) {
```

Comment:
code format? as there seems missing spaces

Reply:
Ran reformatter on all touched files. You will see some changes going in for that now.